src/components/Pad/Pad.js

```diff
@@ -4,36 +4,45 @@ import styled from 'styled-components/macro'
 
 Pad.propTypes = {
   name: PropTypes.string.isRequired,
+  index: PropTypes.number.isRequired,
+  onTouchStart: PropTypes.func.isRequired,
+  onTouchEnd: PropTypes.func.isRequired,
   player: PropTypes.object,
+  isTriggered: PropTypes.bool.isRequired,
 }
 
-export default function Pad({ name, player }) {
+export default function Pad({
+  name,
+  index,
+  onTouchStart,
+  onTouchEnd,
+  player,
+  isTriggered,
+}) {
   return (
     <PadStyled
-      onTouchStart={() => {
-        player.start()
-      }}
-      onTouchEnd={stopPinchZooming}
-      onMouseDown={() => {
-        player.start()
-      }}
+      onTouchStart={() => onTouchStart(index, player)}
```

Comment:
Maybe you can find a way to dispatch on player higher up, preferable to in `padPlayer.js` to reduce coupling?